dhall-lsp-server/src/Dhall/LSP/Handlers.hs

```diff
@@ -267,6 +267,7 @@ diagnosticsHandler uri = do
             _source = Just "Dhall.Lint"
             _code = Nothing
             _message = suggestion
+            _tags = Nothing
```

Comment:
I thought the new field would require tighter lower bounds on `haskell-lsp[-types]`, but then realized that the record wildcards still work with older versions. That's a plus for `-XRecordWildcards` I guess! :)